gpustat/core.py

```diff
@@ -361,15 +408,18 @@ def get_process_info(nv_process):
                 nv_comp_processes = nv_comp_processes or []
                 nv_graphics_processes = nv_graphics_processes or []
                 for nv_process in nv_comp_processes + nv_graphics_processes:
-                    # TODO: could be more information such as system memory
-                    # usage, CPU percentage, create time etc.
                     try:
                         process = get_process_info(nv_process)
                         processes.append(process)
                     except psutil.NoSuchProcess:
                         # TODO: add some reminder for NVML broken context
                         # e.g. nvidia-smi reset  or  reboot the system
                         pass
+                time.sleep(0.1)
```

Comment:
Why do we need a sleep here?

Reply:
Oh I got this.

https://psutil.readthedocs.io/en/latest/index.html#psutil.Process.cpu_percent

When interval is 0.0 or None compares process times to system CPU times elapsed since last call, returning immediately. That means the first time this is called it will return a meaningless 0.0 value which you are supposed to ignore. In this case is recommended for accuracy that this function be called a second time with at least 0.1 seconds between calls.